modules/openapi-generator/src/main/resources/typescript/model/ObjectSerializer.mustache

```diff
@@ -225,6 +225,10 @@ export class ObjectSerializer {
      * Convert data to a string according the given media type
      */
     public static stringify(data: any, mediaType: string): string {
+        if (mediaType === "text/plain") {
+            return data;
```

Comment:
given the return value should be `string`, how about
```suggestion
            return String(data);
```
?

Reply:
My gut feeling is that it would be more proper to "trust" the media-type and surface the error if data is inappropriate.